microservices/users/__tests__/config/remote.ts

```diff
@@ -13,6 +13,7 @@ describe('config/remote', () => {
   it('should correctly return cookies config: with remote', async () => {
     expect(await remoteConfig()).to.deep.equal({
       passwordSaltRounds: CONST.MS_USER_PASSWORD_SALT_ROUNDS,
+      removedAccountRestoreTimeInDays: CONST.MS_USER_REMOVE_ACCOUNT_RESTORE_TIME_IN_DAYS,
```

Comment:
```suggestion
      removedAccountRestoreTime: CONST.MS_USER_REMOVE_ACCOUNT_RESTORE_TIME,
```

and it will be in hours